.circleci/config.yml

```
@@ -85,14 +85,47 @@ jobs:
           path: coverage
 
   test-rendering:
-    executor: browser-node
+    working_directory: ~/dist-flow-charts
+    docker:
+      - image: circleci/node:16.10.0-browsers
+      - image: mcr.microsoft.com/playwright:focal
```

maybe keep 
```suggestion
    executor: browser-node
```
and update browser-node definition (line 8-12)
and then also use it in test-integration